sharktuner/pyproject.toml

```diff
@@ -22,3 +22,8 @@ dynamic = ["version"]
 
 [project.urls]
 Repository = "https://github.com/nod-ai/shark-ai"
+
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["sharktuner*"]
```

Comment:
With this `model_tuner` wont be packages which is, how I understood`, not the intention.